File: lua/refactoring/tests/refactor/123/lua/multiple-vars/inline_var.start.lua
-- stylua: ignore start

local function orderCalculation(order)
    local basePrice, i = order.quantity * order.itemPrice, 3
    return basePrice
        - math.max(0, order.quantity - 500) * order.itemPrice * 0.05
        + math.min(basePrice * 0.1, 100)
end

File: lua/refactoring/tests/refactor/123/go/multiple-vars/inline_var.start.go
package main

import "math"

type Order struct {
    quantity  float64
    itemPrice float64
}

func orderCalculation(order Order) float64 {
    basePrice, i := order.quantity*order.itemPrice, 3
    return basePrice - math.Max(0, order.quantity-500)*order.itemPrice*0.05 + math.Min(basePrice*0.1, 100.0)
}
